decidim-admin/app/queries/decidim/admin/user_groups_evaluation.rb

```diff
@@ -30,11 +33,9 @@ def query
 
       private
 
-      attr_reader :user_group
-
       def filter_by_search(user_groups)
         return user_groups if @q.blank?
-        user_groups.where("LOWER(name) LIKE LOWER('%#{@q}%')")
```

Comment:
OMG :(. Fortunately this only happens on the admin panel.

Reply:
Yeah, I thought the same thing :)